blueprints/APIv3.tex

```diff
@@ -492,9 +585,9 @@ \subsection{Traffic Stats Servers}
 
 \subsection{Traffic Vaults}
 
-\subsection{Users}
+\subsection{Users\label{sec:users}}
 
-\section{Tenantable Objects}
+\section{Tenantable Objects\label{sec:tenantable}}
```

Comment:
I'd also say there should be some objects which are specifically belonging to the root tenant including things such as cachegroups, asns, physical locations, servers, etc for security purposes.

Reply:
Those things don't belong to any tenant today - they aren't tenantable. Are you saying that If I'm on any child tenant then I shouldn't be allowed to see cachegroups, asns, physical locations or servers at all?

Reply:
Correct.  I view those pieces of information as privileged because they could be used to identify weak points in the overall architecture or contain contact information for facilities to be used in spearphishing attacks.

Reply:
I'd be ok if it were just a separate role capability too rather than requiring the root tenant so it could be granted on exception.  That's actually probably a better answer than requiring the root tenant because the root tenant exposes all plaintext configuration on a DS.

Reply:
Also, I'm not on our root tenant, so that'd leave me pretty crippled :P.

Yeah, I don't think Tenancy should be used to limit what _kind_ of objects someone can see/manipulate, only _which_ objects. That's what User Permissions are for.